composer.json

```diff
@@ -77,6 +64,7 @@
         "burdamagazinorg/infinite_module": "^1.0",
         "burdamagazinorg/infinite_theme": "^1.0",
         "composer/installers": "^1.0",
+        "oomphinc/composer-installers-extender": "^1.1",
```

Comment:
I think that ```oomphinc/composer-installers-extender``` package is not need anymore.

Also I see that we have in composer:
```
    "provide": {
        "heiseonline/shariff": "master"
    },
```
and I'm not sure why it's needed?